setup.cfg

```diff
@@ -61,25 +61,22 @@ upload-dir = docs/_build/html
 
 [compile_catalog]
 directory = flask_security/translations/
-use-fuzzy = True
-domain = flask_security
```

Comment:
minor: do you know what domain was supposed to do?

Reply:
I'd say leave it as-is unless it's presenting a particular issue for translations. I don't have a strong opinion, just pointing out that it was a change.